source/draw/gpu/GpuContext.ooc

```diff
@@ -56,6 +56,7 @@ GpuContext: abstract class {
 	drawPoints: abstract func (pointList: VectorList<FloatPoint2D>, transform: FloatTransform3D)
 	drawQuad: abstract func
 	enableBlend: abstract func (blend: Bool)
+	blend: abstract func (factor: Float)
```

Comment:
Then this should not be here. It should be in the backend not in the frontend. The same goes for the other methods as well.